app/controllers/users_controller.rb

```diff
@@ -0,0 +1,9 @@
+class UsersController < ApplicationController
+  before_filter :authenticate_user!
+
+  def show
+    @user = User.find(params[:id])
```

Comment:
Not super sure what you mean here. Explain this to me sometime or show me an example.